system-addon/lib/ThemeFeed.jsm

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+
+ChromeUtils.import("resource://gre/modules/Services.jsm");
+
+const {actionCreators: ac, actionTypes: at} = ChromeUtils.import("resource://activity-stream/common/Actions.jsm", {});
+
+const THEME_UPDATE_EVENT = "lightweight-theme-styling-update";
+
+this.ThemeFeed = class ThemeFeed {
```

Comment:
This feed will be obsolete or need to change once https://bugzilla.mozilla.org/show_bug.cgi?id=1444459 is implemented and lands.